target/linux/octeon/files/arch/mips/boot/dts/cavium-octeon/n821.dts

```diff
@@ -0,0 +1,363 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/dts-v1/;
+
+/ {
+        model = "Cisco/Viptela vEdge 1000";
+        compatible = "n821";
+        #address-cells = <2>;
+        #size-cells = <2>;
+        interrupt-parent = <&ciu>;
+
+        soc {
+                compatible = "simple-bus";
+                #address-cells = <2>;
+                #size-cells = <2>;
+                ranges; /* Direct mapping */
+
+                ciu: interrupt-controller@1070000000000 {
+                        compatible = "cavium,octeon-3860-ciu";
+                        interrupt-controller;
+                        #interrupt-cells = <2>;
+                        reg = <0x10700 0x00 0x00 0x7000>;
+                };
```

Comment:
You should be able to replace most of these nodes by including [octeon_3xxx.dtsi](https://elixir.bootlin.com/linux/v6.1.38/source/arch/mips/boot/dts/cavium-octeon/octeon_3xxx.dtsi) instead.

Look at `cn5020_ubnt_usg.dts` for an example.